src/runtime/crdt/tests/crdt-entity-test.ts

```diff
@@ -39,7 +39,7 @@ describe('CRDTEntity', () => {
     };
     const entity = new CRDTEntity(singletons, collections);
 
-    const value = {id: 'bob', value: 'bob'} as Referenceable;
+    const value = {id: 'bob', value: 'bob'};
```

Comment:
Can we use 
`const value: Referenceable = ...`

Reply:
it's not Referenceable, it's T extends Referenceable.